cmd/cosign/cli/sign.go

```diff
@@ -241,10 +247,31 @@ func SignCmd(ctx context.Context, so SignOpts,
 		}
 		rekorBytes = pemBytes
 	}
-	index, err := cosign.UploadTLog(sig, payload, rekorBytes)
+	entry, err := cosign.UploadTLog(sig, payload, rekorBytes)
 	if err != nil {
 		return err
 	}
-	fmt.Println("tlog entry created with index: ", index)
+	fmt.Println("tlog entry created with index: ", *entry.LogIndex)
+
+	uo.Bundle = bundle(entry)
+	uo.AdditionalAnnotations = annotations(entry)
+	if _, err = cosign.Upload(ctx, uo); err != nil {
+		return errors.Wrap(err, "uploading")
+	}
 	return nil
 }
+
+func bundle(entry *models.LogEntryAnon) *cosign.Bundle {
+	if entry.Verification == nil {
```

Comment:
it's good to check, but is this an expected case? Should we warn here?

Reply:
I put it in there in case people were verifying old signatures that wouldn't have this! That's pretty much the only case I can think of.

Reply:
Ah makes sense.